src/main/java/com/starcallingassist/modules/sidepanel/objects/StarListEntryAttributes.java

```diff
@@ -263,6 +263,10 @@ else if (types.contains(WorldType.FRESH_START_WORLD))
 		{
 			return PluginColors.FRESH_START_WORLD;
 		}
+		else if (types.contains(WorldType.PVP))
+		{
+			return PluginColors.PVP_WORLD;
+		}
 
 		return null;
```

Comment:
The default color is barely visible. We should return some color other than null here. This would make unhandled world types such as `HIGH_RISK` more noticeable.